packages/frontend/src/components/KeyboardShortcutHint.tsx

```diff
@@ -199,9 +199,19 @@ function Shortcut(action: ShortcutAction): CheatSheetEntryType {
 
 export function CheatSheetKeyboardShortcut() {
   if (runtime.getRuntimeInfo().isMac) {
-    return <KeyboardShortcut elements={['Meta', '/']} />
+    return (
+      <>
+        <KeyboardShortcut elements={['Meta', '/']} />
+        <KeyboardShortcut elements={['Meta', '-']} />
+      </>
+    )
   } else {
-    return <KeyboardShortcut elements={['Control', '/']} />
+    return (
+      <>
+        <KeyboardShortcut elements={['Control', '/']} />
+        <KeyboardShortcut elements={['Control', '-']} />
```

Comment:
I don't understand this. `Ctrl + -` doesn't work for me, and I don't see how it should:

https://github.com/deltachat/deltachat-desktop/blob/c176996479646c719883a556a1f5cfc95bcbd31a/packages/frontend/src/keybindings.ts#L144-L146

Reply:
I think it's the german keyboard layout?
When testing here https://developer.mozilla.org/en-US/docs/Web/API/Element/keydown_event with a Linux Notebook, a Macbook Pro or another random german Windows Notebook, the "dash" key laways has the Slash key code. Although Wikipedia mentions the german keyboard has a ÷ sign which triggers the slash which is not the case for mine and (some others) keyboards.
Maybe we do a non-representative survey in Desktop dev channel?

Reply:
Better try it here, where both the key and the code are shown. https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code

I thought `code` is supposed to be layout-independent.

Reply:
Oh. Looks like on the Germain keyboard layout the "dash" key is where the Slash keys is on a US one. I get it now.

![image](https://github.com/user-attachments/assets/428d53cd-c783-4375-9eca-06a1507824b3)

Well, this is a general issue that applies not just to this shortcut and not just to German. We should not add this hint unconditionally.

Reply:
I guess I could be to blame here, because I switched the check from `key` to `code` to ensure that we check the key location and not the printed character, so that the shortcut is independent of the layout. For example, we have `Ctrl + ,` to open the settings dialog, but on the Russian layout a comma is `Shift + Slash` and not `Comma`, so on the Russian layout the shortcut to open settings changes to `Ctrl + Shift + Slash`.

I'm for tackling this in a separate issue.

Reply:
The same in german keyboard layout: to have the key "/" you have to push Ctrl + Shift + /

So I will remove the second Key stroke for opening the shortcut hints.